README.md

```diff
@@ -2,11 +2,38 @@
 
 The Akeneo PIM documentation
 
-## Installation
+## Build the documentation
+
+### With Docker
+
+Install [Docker](https://docs.docker.com/engine/installation/).
+
+[optional] To update the branch list with the current pim-docs branches, use:
+
+```
+sed -i -e "s/^\(.*\)'versions': .*,\(.*\)$/\1'versions': ['$(git branch -l|grep -x "\(^[ *]\+[0-9]\+\.[0-9]\+.*\)\|\(^[ *]\+master\)" | cut -c 3- | sort -r | paste -sd " ")'],\2/" conf.py
+```
+
+From the `./pim-docs` directory, run:
+
+```bash
+    $ docker build . --tag pim-docs:master
+    $ rm -rf pim-docs-build && mkdir pim-docs-build
+    $ docker run --rm \
+        -v $(pwd):/home/akeneo/pim-docs/data \
+        pim-docs:master \
+        ./build.sh master --uid $(id -u) --gid $(id -g) --no-asset-check
```

Comment:
(You removed the no-asset-check)

Reply:
yes, as now we always download the `pim.css`